src/Energy_Preserving.jl

```diff
@@ -0,0 +1,442 @@
+#This code is based on the Theorem 2 of the paper "Energy-Preserving Integrators and the Structure of B-series" (link: https://link.springer.com/article/10.1007/s10208-010-9073-1).
```

Comment:
These new files are never `include`d from the main file `src/BSeries.jl`. Thus, their code is not loaded at all.

Reply:
If this code is moved to `src/BSeries.jl`, you don't need this file anymore, do you?

Reply:
I removed this file.